ironfish/src/storage/database/types.ts

```diff
@@ -4,6 +4,13 @@
 
 import { IJsonSerializable } from '../../serde'
 
+export interface DatabaseIterOptions {
```

Comment:
nit - i think the naming clashes a little with the usage in `StorageUtils.isInRange`, which makes me think this is a range, or "range options"

it seems like this interface is the most general form of `DatabaseKeyRange`, and the interface below is a specific type of range

i don't think changing the name of `DatabaseKeyRange` is in the scope of this pr if we were to do it at all

Reply:
Good call. I'll remove mine, use DatabaseKeyRange, and expand it to have the new fields.